components/input/src/input/input.js

```diff
@@ -187,6 +200,14 @@ export class Input extends Component {
                         'read-only': readOnly,
                     })}
                 />
+                {clearable && value.length ? (
```

Comment:
`value` can be undefined, causing this to crash. 

```suggestion
                {clearable && value?.length ? (
```